.github/workflows/labels.yml

```diff
@@ -5,8 +5,14 @@ on:
   issues:
     types: [labeled]
 
+permissions:
+  contents: read
+
```

Comment:
This one lives in https://github.com/WeblateOrg/meta/ as well (see comment on the top).